pkg/util/k8s/k8s_test.go

```diff
@@ -2574,3 +2577,46 @@ func TestWarningEvent(t *testing.T) {
 	lastEvent := <-recorder.Events
 	require.True(t, strings.Contains(lastEvent, "test reason"))
 }
+
+func TestCreateCRD(t *testing.T) {
+	fakeExtClient := fakeextclient.NewSimpleClientset()
+	apiextensionsops.SetInstance(apiextensionsops.New(fakeExtClient))
+	expectedCRD := &apiextensionsv1.CustomResourceDefinition{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:       "test",
+			Generation: 1,
+		},
+		Spec: apiextensionsv1.CustomResourceDefinitionSpec{
+			Group: "A",
+		},
+	}
+
+	go func() {
+		err := testutil.ActivateCRDWhenCreated(fakeExtClient, expectedCRD.Name)
+		require.NoError(t, err)
+	}()
```

Comment:
nit: I don't think we need this. This is required for other crd registration test because it waits to validate that the CRD has been successfully registered. In our CreateCRD function we are just calling the API and not validating.